daffodil-schematron/README.md

```diff
@@ -25,23 +25,25 @@ Supports standalone sch files or embedded schematron rules within the Daffodil s
 ### Configuration
 
 #### Parameters
+
 - `schematron`: default parameter provides path to the schematron file or resource
 
 #### XSLT & XPATH versions
 
-Schematron defaults to 1.0 of both XSLT and XPath.  Use 2.0 by setting `sch:queryBinding="xslt2"` on the schema.
+Schematron defaults to 1.0 of both XSLT and XPath. Use 2.0 by setting `sch:queryBinding="xslt2"` on the schema.
 
-See version support comments in the [XSL](src/main/resources/iso-schematron-xslt2/iso_schematron_skeleton_for_saxon.xsl) for more details.
+See version support comments in the [XSL](src/main/resources/iso-schematron-xslt2/iso_schematron_skeleton_for_saxon.xsl)
+for more details.
 
 ### Parse Results
 
 - `SchematronValidationError` on failed rule check
 
 ### SPI registration
 
-Registered as `schematron`  
+Registered as `schematron`
 
-`Validators.get("schematron")`
+`Validators.getInstance().get("schematron")`
```

Comment:
I *think* we can get rid of all instances of this getInstance() function except for instance the Validators class.